.gitignore

```diff
@@ -0,0 +1,2 @@
+go.sum
+go.mod
```

Comment:
Try to avoid multiple commits. It's hard to check. You can use git commit --amend option, and git push --force if you have already done PR.